src/main/java/fi/helsinki/cs/tmc/core/communication/TmcServerCommunicationTaskFactory.java

```diff
@@ -76,12 +78,12 @@ public TmcServerCommunicationTaskFactory(TmcSettings settings, Oauth oauth) {
     }
 
     public TmcServerCommunicationTaskFactory(
-            TmcSettings settings,
-            Oauth oauth,
-            AdaptiveExerciseParser adaptiveExerciseParser,
-            CourseListParser courseListParser,
-            CourseInfoParser courseInfoParser,
-            ReviewListParser reviewListParser) {
+                TmcSettings settings,
+                Oauth oauth,
+                AdaptiveExerciseParser adaptiveExerciseParser,
+                CourseListParser courseListParser,
+                CourseInfoParser courseInfoParser,
+                ReviewListParser reviewListParser) {
```

Comment:
Are these changes in indetation intended (I.e., fixing previously incorrect indentation)?